pyomo/contrib/doe/tests/test_doe_solve.py

```diff
@@ -551,15 +553,9 @@ def test_doe_full_factorial(self):
         self.assertStructuredAlmostEqual(
```

Comment:
Is there any reason why these tests specify `abstol` but the new ones below do not? If there is a technical reason, just add a comment.